spec/spec_helper.rb

```diff
@@ -5,11 +5,12 @@ class FeatureSpec < MiniTest::Spec
   require "capybara/poltergeist"
   include Capybara::DSL
   Capybara.register_driver :poltergeist do |config|
-    Capybara::Poltergeist::Driver.new(config, {js_errors: false,
-                                               timeout: 1000,
-                                               phantomjs_options: ['--ignore-ssl-errors=yes',
-                                                                   '--ssl-protocol=any']})
-    # config hash borrowed from 'http://qiita.com/meganii/items/044c606f34918d866a61'
+    Capybara::Poltergeist::Driver.new(
+      config, { js_errors: false,
+                timeout: 1000,
+                phantomjs_options: ['--ignore-ssl-errors=yes',
+                                    '--ssl-protocol=any'] })
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.